spyder/plugins/editor/widgets/tests/test_gotoline.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Tests for gotoline.py"""
+
+# Third party imports
+from qtpy.QtWidgets import QDialogButtonBox, QPushButton, QTableWidget, QLineEdit
```

Comment:
I think the `QPushButton` and `QTableWidget` imports are not being use here so please remove them.